packages/gestalt/src/Box.js

```diff
@@ -199,19 +202,36 @@ const disallowedProps = [
   'lgMarginRight',
 ];
 
-const BoxWithForwardRef: AbstractComponent<Props, HTMLDivElement> = forwardRef<
-  Props,
-  HTMLDivElement,
->(function Box(props, ref): Element<'div'> {
-  const { passthroughProps, propsStyles } = buildStyles<Props>({
-    baseStyles: styles.box,
-    props,
-    blocklistProps: disallowedProps,
-  });
-
-  // And... magic!
-  return <div {...passthroughProps} {...propsStyles} ref={ref} />;
-});
+type OutputType =
+  | Element<'div'>
+  | Element<'article'>
+  | Element<'aside'>
+  | Element<'details'>
+  | Element<'figcaption'>
+  | Element<'figure'>
+  | Element<'footer'>
+  | Element<'header'>
+  | Element<'main'>
+  | Element<'mark'>
+  | Element<'nav'>
+  | Element<'section'>
+  | Element<'summary'>
+  | Element<'time'>;
```

Comment:
@dangerismycat I would suggest to only allow `block` elements for now and add `inline` elements as needed.

E.g. `<Box as="time" padding={4}>` might not do what folks expect. Unless we make `display: block` the default for now.

Reply:
Works for me. Is there a list somewhere of which of these elements are inline vs block?